apps/mobile/src/components/qrcode/Show.tsx

```diff
@@ -1,11 +1,45 @@
+import { Image, StyleSheet } from 'react-native'
 import { SafeAreaView } from 'react-native-safe-area-context'
 
-import { Text } from '../Themed'
+import { Text, View } from '../Themed'
 
 export function Show() {
     return (
         <SafeAreaView>
-            <Text>This is just a placeholder</Text>
+			<View style={styles.centered}>
+				<View style={styles.message}>
+					<Image
+						source={require("../../assets/images/hidetaka.png")}
```

Comment:
We already have the profile picture, instead of showing this avatar, get dynamically it from the profile.